src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -79,7 +94,15 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
         Utils utils;
         line = utils.trim_copy(line, " \f\n\r\t\v");
 
+        if (currentFESession > 1) {
+            canCalibrate = false;
+        } else if (currentFESession == 1) {
+            canCalibrate = true;
+        }
```

Comment:
What about the else case? Ideally we should have something like.

```
if (currentFESession > 1) {
            canCalibrate = false; 
       } 
else if (currentFESession == 1) {
            canCalibrate = true;
        } else {
       //Return error?
}
```
We should have meaningful statement in the above else clause rather than a comment.